CHANGELOG.md

```diff
@@ -11,6 +11,42 @@
 - [go-mod-registry](https://github.com/edgexfoundry/go-mod-registry/blob/main/CHANGELOG.md) 
 - [go-mod-configuration](https://github.com/edgexfoundry/go-mod-configuration/blob/main/CHANGELOG.md)
 
+## [v3.1.0] Napa - 2023-11-15 (Only compatible with the 3.x releases)
+
+
+### ✨  Features
+
+- Remove snap packaging ([6eeb2ce…](https://github.com/edgexfoundry/edgex-ui-go/commit/6eeb2ce6fc2afc5ffacdf29fd0663685f6ab0964))
+```text
+
+BREAKING CHANGE: Remove snap packaging
+
+```
+- New Provision Watchers tab
+  - Add Provision Watchers tab to Metadata ([a321165…](https://github.com/edgexfoundry/edgex-ui-go/commit/a321165113bd0bfa455e6042dab80b5a852e9ebb))
+  - Fields for Provision Watcher Tab DTO ([f7e57eb…](https://github.com/edgexfoundry/edgex-ui-go/commit/f7e57ebd2ad4d96da9d70341491ca8456df46195))
+  - More for Add Provision Watchers tab to Metadata ([e723d52…](https://github.com/edgexfoundry/edgex-ui-go/commit/e723d524dc2486b7380e56c06e54ac33686a11d2))
```

Comment:
Is this the right wording? or should it be something else? It's a little awkward as is.

```suggestion
  - Add Metadata for Add Provision Watchers tab ([e723d52…](https://github.com/edgexfoundry/edgex-ui-go/commit/e723d524dc2486b7380e56c06e54ac33686a11d2))
```

Reply:
I changed is so it wasn't duplicate of first one. How about `Additional Add Provision Watchers tab to Metadata`?

Reply:
Additional changes for Add...